config/base_config.json

```diff
@@ -13,6 +13,5 @@
   "powerBiUrl": "",
   "gaId": "",
   "features": {
-    "newChairConsole": false
```

Comment:
The values in these configs should somewhat align. I recommend that we add the `"newDarUi": true` as well so other configs are consistent.